### ISSUE TITLE
HAR Output quietly fails when using an existing flow file

### ISSUE THREAD

**Issue by ironsalsa** [mitmproxy/mitmproxy#3240]
TL;DR How to output HAR from existing flow

##### Steps to reproduce the problem:

1. Download https://github.com/mitmproxy/mitmproxy/blob/ad74c18f74256537fd22037e8aec0e9707773fa0/examples/complex/har_dump.py

2. mitmdump -ns ./har_dump.py -r /trash/flowsinnativeformat --set hardump=/trash/hardump.har

Output:
Loading script /trash/har_dump.py
192.168.13.203:51406: POST http://23.101.136.37/track
                   << 200 OK 2b
192.168.13.203:32791: POST http://172.82.210.19/b/ss/nbcusportslive/0/JAVA-4.6.0-AN/s55284490
                   << 200 OK 1b
etc. etc.

The flow loads, but no HAR output is ever generated - is there a way to load an MITMdump file and convert that to HAR

##### Any other comments? What have you tried so far?
There used to be a har_extractor.py around, but that is now 404'd when I find references to it.

##### System information

$ docker run -it mitmproxy/mitmproxy mitmdump --version
Mitmproxy: 4.0.1
Python:    3.6.3
OpenSSL:   OpenSSL 1.0.2o  27 Mar 2018
Platform:  Linux-4.15.0-24-generic-x86_64-with

**Comment by ironsalsa** [comment]
Does this have to do with the def done() line in har_dump.py? I'm wondering if mitmdump -n doesn't send a done() signal?

**Comment by ironsalsa** [comment]
The issue is that I was running commands from a different folder than the output - once I switched directories to where I was outputting, this was no longer an issue.
